cdc-sandbox/db/Dockerfile

```diff
@@ -12,9 +12,10 @@ USER mssql
 ADD ./restore /var/opt/db-restore/
 
 # docker-compose environment variables are only visible at runtime
+ARG DB_PW
```

Comment:
This argument is something that must be passed in whenever the container is built, correct? Has this been tested with the GitHub pipeline? We may need to modify how containers are built.

Reply:
ah, yes. Need to update the `Build-modernization-test-db.yaml`